addons/velaux/resources/velaux.cue

```diff
@@ -23,24 +23,26 @@ output: {
 		if parameter["serviceType"] != _|_ {
 			exposeType: parameter["serviceType"]
 		}
-		if parameter["dex"] == "true" {
-			env: [
-				{
-					name:  "KUBEVELA_API_URL"
-					value: "apiserver.vela-system:8000"
-				},
-				{
-					name:  "DEX_URL"
-					value: "dex.vela-system:5556"
-				},
-			]
+		if parameter["dex"] != _|_ {
+			if parameter["dex"] == true {
```

Comment:
What if `dex=false`? In this case, `KUBEVELA_API_URL` won't be set.

Reply:
Yes that makes more sense, thanks for the review.